client_internals_it_test.go

```diff
@@ -49,28 +49,28 @@ func TestListenersAfterClientDisconnected(t *testing.T) {
 		testListenersAfterClientDisconnected(t, "localhost", "127.0.0.1", 46501, addEntryListener)
 	})
 	t.Run("MemberHostname_ClientIP_AddListener", func(t *testing.T) {
-		testListenersAfterClientDisconnected(t, "localhost", "127.0.0.1", 46501, addListener)
+		testListenersAfterClientDisconnected(t, "localhost", "127.0.0.1", 46502, addListener)
```

Comment:
Could you revert the port changes in this file? They don't seem to be helping.

Reply:
Fixed: [88b6f60](https://github.com/hazelcast/hazelcast-go-client/pull/809/commits/88b6f60a915c3d0237ae5f1fb97bdbda52cedfbd)